src/sdk/model/org/org.ts

```diff
@@ -514,12 +516,12 @@ export class Org extends Entity {
 
   /**
    * Gets all DNS zones that exist within the org.
-   * @returns {Promise<Array<DnsZone>>} a promise that resolves with the list of DNS zones
+   * @returns {Promise<Array<OrgDnsZone>>} a promise that resolves with the list of DNS zones
    */
-  async getDnsZones(): Promise<Array<DnsZone>> {
+  async getDnsZones(): Promise<Array<OrgDnsZone>> {
```

Comment:
Can rename this to getOrgDnsZones to match the return type update. Is the original DnsZone used by anything else?

Reply:
yes `DnsZone` is returned by `Org.addDnsZone()`

Reply:
Thanks, I see that this matches the API now.